lib/evaluateFunctions.js

```diff
@@ -96,4 +99,389 @@ function absoluteValue(node) {
   }
 }
 
+// Evaluate nthRoot() function.
+// Returns a NodeStatus object.
+function nthRoot(node) {
```

Comment:
Yea, I discuss re-organizing modules in the mathsteps organization doc, let's discuss that more